BlueDB/src/main/java/io/bluedb/api/CloseableIterator.java

```diff
@@ -0,0 +1,7 @@
+package io.bluedb.api;
+
+import java.io.Closeable;
+import java.util.Iterator;
+
+public interface CloseableIterator<E> extends Closeable, Iterator<E> {

```

Comment:
How do you decide what letter to use in your generic templates? For example `E` or `X` instead of `T`. I've always just used `T`. I'm just curious.

Reply:
Eclipse automatically populated Iterator with E (presumable Element?) so I went with it.  Mostly I use T or for a generic static factory method, I use X.